docs/pages/choose-an-edition/teleport-team.mdx

```diff
@@ -4,17 +4,19 @@ description: "Teleport Team makes it quick and easy for small organizations set
 tocDepth: 3
 ---
 
-Teleport Team helps small organizations get started quickly enabling secure
-access to their infrastructure. 
+Teleport Team helps small organizations provide secure access to their infrastructure, with 
+minimal configuration and no management overhead. 
 
-With Teleport Team, the Auth Service and Proxy Service are fully managed, giving
-you a scalable, fault tolerant certificate authority and reverse proxy that you
-can use to configure access from users across the globe to your infrastructure,
-including private networks.
+With Teleport Team, the Teleport Auth Service and Teleport Proxy Service are professionally 
+managed in the cloud. By connecting to these services in the cloud, you have immediate access
+to a scalable and fault tolerant certificate authority and reverse proxy that you don't need 
+to manage or maintain. Instead of managing the infrastructure, you can focus on adding users 
+and resources you want to protect and configuring secure role-based access for private and public
```

Comment:
```suggestion
to manage or maintain. You can focus on enrolling resources you want to protect and configuring secure role-based access for private and public
```
Since you wouldn't be protecting users but infra resources